cmd/server.go

```diff
@@ -195,15 +195,23 @@ func NewServerCommand(stopCh <-chan struct{}) *cobra.Command {
 					}
 					if err != nil {
 						if etcdErr, ok := err.(*errors.EtcdError); ok == true {
```

Comment:
I don't find it making sense to have different log messages here. Let's just combine it.  Remove typecasting and if-else and have single log message on error.

Reply:
Sure. I kept the error messages distinct because the same convention was followed here https://github.com/gardener/etcd-backup-restore/blob/8f32a9a72405589ae6a5d4d4192d002e1adb64f6/cmd/server.go#L221

Reply:
Agreed. But earlier we used to call logger.Fatalf for non etcderr / snapshot upload related error. I don;t remeber why we changed that. But now since you have pointed that out, it should be `Fatalf` at https://github.com/gardener/etcd-backup-restore/blob/8f32a9a72405589ae6a5d4d4192d002e1adb64f6/cmd/server.go#L224.

Reply:
Ah ok. I'll make that change then.